src/docs/getting-started/quick-start.md

```diff
@@ -1,72 +1,92 @@
-izm5ehdjw3kru84f0kq7r8z Ready master 18h v1.19.8
-izm5ehdjw3kru84f0kq7r9z Ready <none> 18h v1.19.8
-izm5ehdjw3kru84f0kq7raz Ready <none> 18h v1.19.8
-izm5ehdjw3kru84f0kq7rbz Ready <none> 18h v1.19.8
+izm5e42unzb79kod55hehvz Ready master 18h v1.22.15
+izm5ehdjw3kru84f0kq7r7z Ready master 18h v1.22.15
+izm5ehdjw3kru84f0kq7r8z Ready master 18h v1.22.15
+izm5ehdjw3kru84f0kq7r9z Ready <none> 18h v1.22.15
+izm5ehdjw3kru84f0kq7raz Ready <none> 18h v1.22.15
+izm5ehdjw3kru84f0kq7rbz Ready <none> 18h v1.22.15
 ```
 
-## Clean the cluster
-
-Some information of the basic settings will be written to the Clusterfile and stored in /root/.sealer/[cluster-name]/Clusterfile.
-
-```shell script
-sealer delete -f /root/.sealer/my-cluster/Clusterfile
+## Build an app image
+
+nginx.yaml:
+
+```shell
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-nginx
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: my-nginx
+  template:
+    metadata:
+      labels:
+        run: my-nginx
+    spec:
+      containers:
+        - name: my-nginx
+          image: nginx
+          ports:
+            - containerPort: 80
 ```
 
-## Build your own CloudImage
-
-For example, build a dashboard CloudImage:
-
 Kubefile:
 
-```shell script
-# base CloudImage contains all the files that run a kubernetes cluster needed.
-#    1. kubernetes components like kubectl kubeadm kubelet and apiserver images ...
-#    2. docker engine, and a private registry
-#    3. config files, yaml, static files, scripts ...
-FROM registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.19.8
-# download kubernetes dashboard yaml file
-RUN wget https://raw.githubusercontent.com/kubernetes/dashboard/v2.2.0/aio/deploy/recommended.yaml
-# when run this CloudImage, will apply a dashboard manifests
-CMD kubectl apply -f recommended.yaml
+```shell
+FROM scratch
+APP nginx local://nginx.yaml
+LAUNCH ["nginx"]
 ```
 
-Build dashboard CloudImage:
+```shell
+sealer build -f Kubefile -t sealer-io/nginx:latest --type app-installer
+```
+## Run the app image
 
-```shell script
-sealer build -t registry.cn-qingdao.aliyuncs.com/sealer-io/dashboard:latest .
+```shell
+sealer run sealer-io/nginx:latest
+# check the pod
+kubectl get pod -A
 ```
 
-Run your kubernetes cluster with dashboard:
+## Push the app image to the registry
 
-```shell script
-# sealer will install a kubernetes on host 192.168.0.2 then apply the dashboard manifests
-sealer run registry.cn-qingdao.aliyuncs.com/sealer-io/dashboard:latest --masters 192.168.0.2 --passwd xxx
-# check the pod
-kubectl get pod -A|grep dashboard
+```shell
+# you can push the app image to docker hub, Ali ACR, or Harbor
+sealer tag sealer-io/nginx:latest {registryDomain}/sealer-io/nginx:latest
+sealer push {registryDomain}/sealer-io/nginx:latest
 ```
 
-## Push the CloudImage to the registry
+## Clean the cluster
+
+Some information of the basic settings will be written to the Clusterfile and stored in /root/.sealer/Clusterfile.
```

Comment:
Some information of the basic settings will be written to the cluster and stored in /root/.sealer/Clusterfile locally.